sched/test_hash_stress.py

```diff
@@ -34,17 +36,34 @@ def configure_tempesta(self):
             sg.sched = 'hash'
 
     def assert_servers(self):
+        reqs_exp = self.tempesta.stats.cl_msg_received
         self.servers_get_stats()
         # Only one server must pull all the load.
```

Comment:
It seems that comment should be changed or removed, since not `only` one server pull all the load in test.

Reply:
Fixed.